app/presenters/content_item/recruitment_banner.rb

```diff
@@ -0,0 +1,18 @@
+module ContentItem
+  module RecruitmentBanner
+    BRAND_SURVEY_URL = "https://gov.uk/".freeze
+    SURVEY_URL_MAPPINGS = {
+      "/check-national-insurance-record" => BRAND_SURVEY_URL,
+      "/check-mot-history" => BRAND_SURVEY_URL,
+    }.freeze
+
+    def recruitment_survey_url
+      user_research_test_url
+    end
+
+    def user_research_test_url
```

Comment:
Could we call this `brand_user_research_test_url` the same as you've done in[ government frontend](https://github.com/alphagov/government-frontend/pull/2932/files#diff-cb70056828d7a1821457717b78e2d07c7932af21af7e3de4dde362b7db5719f2R14) so we don't get confused when we have to extend this module with the AI banner 🙃